test/src/device-request-prompt.spec.ts

```diff
@@ -19,9 +19,11 @@ import {TimeoutError} from 'puppeteer';
 import {launch} from './mocha-utils.js';
 
 describe('device request prompt', function () {
+  this.timeout(1_000);
   let state: Awaited<ReturnType<typeof launch>>;
 
   before(async () => {
```

Comment:
Else the `this` is not correct.
```suggestion
  before(async function () {
```